pipelines/_ingredients/ApPipe.yaml

```diff
@@ -163,7 +163,7 @@ contracts:
   - contract: diaPipe.connections.ConnectionsClass(config=diaPipe).associatedDiaSources.name ==
               analyzeAssocDiaSrcCore.connections.ConnectionsClass(config=analyzeAssocDiaSrcCore).data.name
     msg: "diaPipe.associatedDiaSources != analyzeAssocDiaSrcCore.data"
-  - contract: diaPipe.connections.ConnectionsClass(config=diaPipe).longTrailedSources.name ==
+  - contract: filterDiaSrcCat.connections.ConnectionsClass(config=filterDiaSrcCat).longTrailedSources.name ==
               analyzeTrailedDiaSrcCore.connections.ConnectionsClass(config=analyzeTrailedDiaSrcCore).data.name
     msg: "diaPipe.longTrailedSources != analyzeTrailedDiaSrcCore.data"
```

Comment:
Please keep the contracts sorted (see comment at the top of list) so that they're easier to find/check later. Also, the message no longer matches the connections.

Reply:
Updated the message and moved the contract up beneath the other filterDiaSourceCatalog contracts.